app/android/app/src/main/AndroidManifest.xml

```diff
@@ -26,7 +26,17 @@
         android:icon="@mipmap/ic_launcher"
         android:requestLegacyExternalStorage="true"
     >
-
+        <provider
```

Comment:
Is this needed? I believe the app worked before also

Reply:
No it wasn't needed . Actually i got some storage error with one of my devices so i thought it might be the issue. I will remove it